docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -283,7 +283,158 @@ kube-dns-{network} shall be managed by Endpoints controller just like a regular
 
 __EndpointSlices__, the new type introduced in k8s v1.17, is out of current scope.
 
-### Ingress/egress/network policies
+### Network Policy
+
+Network policies, expressed as intents, are used to enforce desired network security within Arktos. Without any network policy in place, Arktos allows any pod to access other pods, across all tenants and namespaces for a flat typed network environment. VPC typed network environment, on the other hand, does provide strong tenant level network security even in the absence of any network policy in place.
```

Comment:
Not sure why we emphasize "expressed as intents" here.  My understanding is all objects created by users in K8S/Arktos are expressed as intents. Is there something special here?

Reply:
network policies by themselves do nothing; they rely on policy rule enforcer to achieve the desired network access security. Since it caused confusion, I'll remove this sentence.